src/main/java/de/cas/mse/exercise/diamond/Diamond.java

```diff
@@ -4,28 +4,28 @@
 
 public class Diamond {
 
-	public String print(int n) {
-		if (n <= 0 || n % 2 == 0) {
+	public String createDiamond(int maximumDimension) {
+		if (maximumDimension <= 0 || maximumDimension % 2 == 0) {
```

Comment:
Consider moving this check into an isValidDiamond() method just to be self-documenting.